diff --git a/CMakeLists.txt b/CMakeLists.txt
index 044a2de8a..480558a34 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -182,23 +182,6 @@ endmacro()
 jerry_add_compile_flags(${FLAGS_COMMON_ARCH})
 jerry_add_flags(CMAKE_EXE_LINKER_FLAGS ${FLAGS_COMMON_ARCH})
 
-# LTO
-if(ENABLE_LTO)
-  if(USING_GCC OR USING_CLANG)
-    jerry_add_compile_flags(-flto)
-    jerry_add_link_flags(-flto)
-  endif()
-  if(USING_GCC)
-    jerry_add_compile_flags(-fno-fat-lto-objects)
-    # Use gcc-ar and gcc-ranlib to support LTO
-    set(CMAKE_AR "gcc-ar")
-    set(CMAKE_RANLIB "gcc-ranlib")
-  endif()
-  if(USING_TI)
-    jerry_add_link_flags(-lto)
-  endif()
-endif()
-
 # Compiler / Linker flags
 if("${PLATFORM}" STREQUAL "DARWIN")
   jerry_add_link_flags(-lSystem)
